.github/draft-release-notes-config.yml

```diff
@@ -0,0 +1,29 @@
+# The overall template of the release notes
+template: |
+  Open Distro for Elasticsearch Version $RESOLVED_VERSION
+  $CHANGES
+
+# Setting the formatting and sorting for the release notes body
+name-template: Version $RESOLVED_VERSION
+change-template: "- $TITLE (PR [#$NUMBER](https://github.com/opendistro-for-elasticsearch/sql-cli/pull/$NUMBER))"
+sort-by: merged_at
+sort-direction: ascending
+
+# Organizing the tagged PRs into categories
+categories:
+  - title: "Version Upgrades"
+    labels:
```

Comment:
What if a PR has multiple labels, e.g. 'enhancement' and 'windows compatibility'. Will this script be able to pick it up and put it under the enhancement title?

Reply:
Good question, I tested on my fork branch. Since there is no 'windows compatibility' label defined in the yml file, it will only be put under 'enhancement'. 
If there is a different category in yml file with label 'windows compatibility', it will be put under both categories. 
If it does not have labels or all labels are undefined in yml file, it will be put at the very top uncategorized.